### ISSUE TITLE
deployment of 'unstable' branch to Azure web app fails

### ISSUE THREAD

**Issue by pacharanero** [rcpch/digital-growth-charts-flask-client#9]
I've been trying to deploy the unstable branch to its own Azure webapp but it fails with a vague Azure-side error. 
I have raised a support ticket with Azure.

**Comment by pacharanero** [comment]
Fixed with Azure support ticket

**Comment by pacharanero** [comment]
For the benefit of anyone else googling this error in the future and finding not much help on the interwebs...

The only error in the Azure UI was: "Failed to set up deployment". Pretty unhelpful.
![image](https://user-images.githubusercontent.com/2348385/95681648-fc612b00-0bd8-11eb-9d8b-90667fcec18a.png)

The JSON error in the Activity Log (also this error was visible in the browser console) was:
"Message\":\"The api is not allowed for subscription (67e313f5-79fe-4480-b867-xxhxhxhxhhx) with co-admins.\"

Curiously I had been able to deploy 4 other webapps (all of our `alpha` suite) without problems, using exactly the same process. But after 4 successful deploys, all subsequent attempts failed. We were deploying a Github repo via Github Actions to an Azure web application / App Service.

I'm not an Azure 'native' - I'm only using it because the web infrastructure of the organisation I'm working with is already on Azure. So I didn't know about 'co-admins' which is apparently part of 'Classic Administrators' - a legacy part of Azure, which has been replaced by a more flexible system of Role-based access control. When I went digging, it looked like we had some co-admins, although I don't think those access controls were being used for anything.

Removing our co-administrators from Azure fixed the problem, and we were able to deploy as normal.
